master/templates/admin/master/city/change_form.html

```diff
@@ -50,89 +43,43 @@
 {% if save_on_top %}{% block submit_buttons_top %}{% submit_row %}{% endblock %}{% endif %}
 {% if errors %}
     <p class="errornote">
-    {% if errors|length == 1 %}{% trans "go to hell." %}{% else %}{% trans "Go To Hell." %}{% endif %}
+    {% if errors|length == 1 %}{% trans "Please correct the error below." %}{% else %}{% trans "Please correct the error below." %}{% endif %}
     </p>
     {{ adminform.form.non_field_errors }}
 {% endif %}
 
 <script>
 function myFunction(){
-	var cityname = django.jQuery("#id_name option:selected").text();
-	console.log(cityname);
-	alert(cityname);
-
-        var id = django.jQuery("#id_name option:selected").val();
-        alert(id);
-        
-	
-
-django.jQuery.ajax({
- 
-    url: "http://127.0.0.1:8000/admin/search/",
- 
-    data: {
-        id : id
-    },
- 
-    // Whether this is a POST or GET request
-    type: "POST",
- 
-    // The type of data we expect back
-    contenttype : "json",
-
-  // Code to run if the request succeeds (is done);
-  // The response is passed to the function
-  
-
-  success :function(json) {
-     
-    alert( "The request is complete!" );
-    alert(json.district_code);
-    console.log(json.district_code);
-   
-
-      
-       django.jQuery('#id_city_code').val(json.city_code);
-       
-       
-      django.jQuery('#id_state_code').val(json.state_code);
-       
-       
-       django.jQuery('#id_state_name').val(json.state_name);
-
-      
-       django.jQuery('#id_district_name').val(json.district_name);
-       
-
-       django.jQuery('#id_district_code').val(json.district_code);
-
-
-       django.jQuery("#id_city_code").prop("readonly", true);
-
-       django.jQuery('#id_state_name').prop("readonly", true);
-      
-       django.jQuery('#id_state_code').prop("readonly", true);
-      
-       django.jQuery('#id_district_name').prop("readonly", true);
-       
-       django.jQuery('#id_district_code').prop("readonly", true);
+    var cityname = django.jQuery("#id_name option:selected").text();
+    var id = django.jQuery("#id_name option:selected").val();
+    django.jQuery.ajax({
+        url : "/admin/search/",
```

Comment:
Do not hard code the url. Instead check Django's url template tag (docs https://docs.djangoproject.com/en/dev/ref/templates/builtins/#url) with your named url pattern.